app/src/main/java/com/weatherxm/data/bluetooth/BluetoothConnectionManager.kt

```diff
@@ -366,4 +368,29 @@ class BluetoothConnectionManager(
             }.collect()
         }
     }
+
+    /**
+     * Custom fix needed for firmware versions < 2.9.0 where ATZ commands
+     * does NOT return an OK before rebooting so we return immediately here
+     */
+    suspend fun reboot(command: String, listener: (Either<Failure, Unit>) -> Unit) {
```

Comment:
Since the function name is `reboot`, passing the reboot command as an argument  seems redundant